dependencies/vaticle/artifacts.bzl

```diff
@@ -25,5 +25,5 @@ def vaticle_typedb_artifact():
         artifact_name = "typedb-server-{platform}-{version}.{ext}",
         tag_source = deployment["artifact.release"],
         commit_source = deployment["artifact.snapshot"],
-        tag = "2.14.2",
```

Comment:
does studio test against cluster at all?

Reply:
Not currently, no.